### DISCUSSION TITLE
Can ovn be changed to a third-party plugin, such as kube-ovn

### DISCUSSION THREAD

**Discussion by zzzzzhy** [okd-project/okd#1613] [General]
This is a usage problem, not a bug,
I want to use kube-ovn to replace okd's native ovn, is it feasible?

If possible, how should it be done?

**Comment by vrutkovs** [comment]
Network operator can be disabled and additional manifests can be provided to replace OVNKubernetes, see [Cilium example](https://docs.cilium.io/en/v1.9/gettingstarted/k8s-install-openshift-okd/)
